plugin/evm/customheader/customheadertest/test_min_delay_excess.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package customheadertest
+
+import "github.com/ava-labs/avalanchego/vms/evm/acp226"
+
+func NewDelayExcessPtr(value uint64) *acp226.DelayExcess {
+	delayExcess := acp226.DelayExcess(value)
+	return &delayExcess
+}
```

Comment:
Can we unify all of these implementations?
- https://github.com/ava-labs/coreth/blob/0a6ce77563f1865b0a05f8d507bb6a4ba222aa26/plugin/evm/customtypes/header_ext_test.go#L192
- https://github.com/ava-labs/coreth/blob/0a6ce77563f1865b0a05f8d507bb6a4ba222aa26/utils/numbers.go#L11

I think a single:
```golang
func PointerTo[T any](x T) *T {
	return &x
}
```
should work for all our usages.